jbpm-container-test/jbpm-remote-ejb-test/jbpm-remote-ejb-test-suite/pom.xml

```diff
@@ -463,7 +463,35 @@
         </pluginManagement>
       </build>
     </profile>
-
+    <profile>
+      <id>custom-container</id>
+      <activation>
+        <!-- The tests are not executed unless specific container profile is used. Disable cargo by default. -->
+        <activeByDefault>true</activeByDefault>
+        <property>
+          <name>custom-container</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.cargo</groupId>
+            <artifactId>cargo-maven2-plugin</artifactId>
+            <executions>
+              <!-- do nothing, container is not managed by cargo -->
+              <execution>
+                <id>start-container</id>
+                <phase>none</phase>
+              </execution>
+              <execution>
+                <id>stop-container</id>
+                <phase>none</phase>
+              </execution>
```

Comment:
I know this is what I originally did in kie-server tests, but I believe nowadays there is a better way. Simple setting property `cargo.maven.skip` to `true` will disable all Cargo related executions (which we want in this case). WDYT?